cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -0,0 +1,87 @@
+const SAFE = 'gor:0x04f8b1EA3cBB315b87ced0E32deb5a43cC151a91'
+const EOA = '0xE297437d6b53890cbf004e401F3acc67c8b39665'
+
+describe('Queue a transaction on N/1', () => {
+  before(() => {
+    cy.connectE2EWallet()
+
+    cy.visit(`/${SAFE}/home`, { failOnStatusCode: false })
+
+    cy.contains('Accept selection').click()
+  })
+
+  it('should create and queue a transaction', () => {
+    // Open the new transaction modal
+    cy.contains('New transaction', { timeout: 10000 }).click()
+
+    // Modal is open
+    cy.contains('h2', 'New transaction').should('be.visible')
+    cy.contains('Send tokens').click()
+
+    // Fill transaction data
+    cy.get('input[name="recipient"]').type(EOA)
+    // Click on the Token selector
+    cy.get('input[name="tokenAddress"]').prev().click()
+    cy.get('ul[role="listbox"]').contains('Görli Ether').click()
+
+    // Insert amount
+    cy.get('input[name="amount"]').type('0.00003')
+
+    cy.contains('Next').click()
+  })
+
+  it('should create a queued transaction', () => {
+    // Alias for New transaction modal
+    cy.contains('h2', 'Review transaction').parents('div').as('modal')
+
+    cy.contains('Signing the transaction with nonce 4').click()
```

Comment:
`4` will be 5, 6, 7, etc in the next test runs.

Reply:
The test keeps queuing with nonce 3 for the reason I explained in the previous comment.

Reply:
I don't understand how your other comment is related. The dashboard displays the oldest txns, I get that. How does the nonce of a new tx depend on that?

Reply:
Because all the test is doing is rewriting the tx with the nonce 3.

Reply:
I see, thanks.

Reply:
I mean, I've taken this approach of not increasing the queued nonce so the test could keep veryfying the Pending queued on the Dashboard